src/themes/eui-amsterdam/overrides/_code_block.scss

```
@@ -0,0 +1,8 @@
+.euiCodeBlock.euiCodeBlock--inline {
+  border-radius: $euiBorderRadiusSmall;
+  color: makeHighContrastColor($euiColorVis3, $euiCodeBlockBackgroundColor);
```

Maybe use a code variable that is the same?
```suggestion
  color: $euiCodeBlockKeywordColor;
```

---

Gonna leave this. I prefer using the root variable until there is more than one usage that exists.